Automattic/jetpack
# CLI - Add Autotagger to Generate

Fixes https://github.com/Automattic/jetpack/issues/19816

#### Changes proposed in this Pull Request:

* adds prompt for autotagger when creating a mirror repo using `jetpack generate`.

#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/19777#discussion_r629921587

#### Does this pull request change what data or activity we track or use?


no 

#### Testing instructions:






* Run `jetpack generate`
* Select the option to create a monorepo
* Select the option to enable autotagger
* Check the new project's composer.json file and see that `extra.autotagger` exists.
